src/foundry/client/documents/tile.d.mts

```diff
@@ -253,7 +253,7 @@ declare namespace TileDocument {
        */
       mode: fields.NumberField<
         {
-          choices: CONST.OCCLUSION_MODES[];
+          choices: Record<CONST.OCCLUSION_MODES, string>;
```

Comment:
Hmm, did this break any tests? Could you rerun tests using `npm run betterer --update` and commit up the results?